core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -2432,6 +2432,10 @@ private[spark] object Utils extends Logging {
   }
 }
 
+private[spark] object CallerContext {
+  var callerContextSupported: Boolean = true
```

Comment:
@srowen I updated the code to make `callerContextSupported` a static variable because the usage pattern of `CallerContext` is `new CallerContext(...).setCurrentContext()`